TraditionalBridge.UnitTest/Dispatcher.dbl

```diff
@@ -5,30 +5,308 @@ import System.Text.Json
+        {TestMethod}
+        public method AutoTimeALin, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatchAutoTime('{"jsonrpc":"2.0","id":12,"method":"AutoTimeALin","params":[{"ReturnedValue":false},{"PassedValue":{"Remainder":"Remainder"},"DataType":1024}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method AutoTimeALout, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatchAutoTime('{"jsonrpc":"2.0","id":13,"method":"AutoTimeALout","params":[{"ReturnedValue":false},{"PassedValue":["","","","",""],"DataType":1024}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method AutoTimeArray, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatchAutoTime('{"jsonrpc":"2.0","id":14,"method":"AutoTimeArray","params":[{"ReturnedValue":false},{"PassedValue":["","",""],"DataType":1024}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method AutoTimeInOut, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatchAutoTime('{"jsonrpc":"2.0","id":15,"method":"AutoTimeINOUT","params":[{"ReturnedValue":false},{"PassedValue":{"GRFA":"","Value":""},"DataType":16}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method AutoTimeIn, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatchAutoTime('{"jsonrpc":"2.0","id":16,"method":"AutoTimeIn","params":[{"ReturnedValue":false},{"PassedValue":{"GRFA":"","Value":""},"DataType":16}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+        {TestMethod}
+        public method AutoTimeOut, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatchAutoTime('{"jsonrpc":"2.0","id":17,"method":"AutoTimeOut","params":[{"ReturnedValue":false},{"PassedValue":{"GRFA":"","Value":""},"DataType":16}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+.endregion
+
+.region "Encrypt"
+        {TestMethod}
+        public method encrypt_tst1, void
```

Comment:
These `encypt_tst#` are conversions from other tests, yeah? Is there any way to give more context about what makes the tests unique? If I saw that encypt_tst10 was failing in a test run, I wouldn't have any clue about what that was.

Reply:
added summaries around each test